utils/api.js

```diff
@@ -38,6 +38,35 @@ export function requestSuggestedNodes() {
   }).then(response => response.data)
 }
 
+export function requestNodeScores(chain, network) {
+  const chainMap = {
+    bitcoin: 'btc',
+    litecoin: 'ltc',
+  }
+  const networkMap = {
+    mainnet: '',
+    testnet: 'testnet',
+  }
+  const code = chainMap[chain]
+  const suffix = networkMap[network]
+  const chainCode = `${code}${suffix}`
+  const BASE_URL = `https://nodes.lightning.computer/availability/v1/${chainCode}.json`
+  return axios({
+    method: 'get',
+    url: BASE_URL,
+  }).then(response => {
+    const result = {}
+    response.data.scores.reduce((map, { public_key, score }) => {
+      if (typeof public_key !== 'string' || !Number.isInteger(score)) {
+        throw new Error('Invalid node score format!')
```

Comment:
Do we really want to discard the entire process if one entry is invalid? Maybe just skip the entry ?

Reply:
Good point. I've changed this so that we log a warning message and continue with the reducer.